judgments/tests.py

```diff
@@ -116,17 +116,42 @@ def test_can_parse_judgment(self):
 
 
 class TestPaginator(TestCase):
-    def test_paginator(self):
+    def test_paginator_2500(self):
```

Comment:
This is the smallest of typos but it should be `250` not `2500`

Reply:
It's the number of judgments, not the number of pages, in each case.